### ISSUE TITLE
Mount folder with command line returns error

### ISSUE THREAD

**Issue by Forusim** [dvdforge/imgdrive_translations#48]
First of all, thanks for this tool.
I was especially looking for the "Mount Folder" feature, which is really rare.

However I was kind of disappointed, that this is feature is not working via command line.
Is there some special undocumented systax or it is just not implemented yet?
https://www.yubsoft.com/imgdrive/index.htm#command_line

**Comment by z3ugma** [comment]
Also finding that attempting to mount folder via command line isn't working. 
Tried:
```
## Unmount first 
PS C:\Program Files\ImgDrive> .\imgdrive.exe -u
##Try to mount folder with f switch
PS C:\Program Files\ImgDrive> .\imgdrive.exe -f C:\Data\Folder
```

**Comment by imgdrive** [comment]
had fixed in v1.7.3

imgdrive.exe /m <folder>

**Comment by imgdrive** [comment]
fixed
